nodejs/week1/src/backend/app.js

```diff
@@ -6,46 +6,46 @@ const reviews = require("./data/reviews.json");
 const app = express();
 
 app.get("/", async (req, res) => {
-    res.json("Meal Sharing Web App");
+    res.send("Meal Sharing Web App");
 });
 
 app.get("/meals", async (req, res) => {
 
-    res.json(module.exports = {meals,reviews});
+    res.send(module.exports = {meals,reviews});
```

Comment:
did you run the tests? i think you missed the part where you were supposed to add the reviews to the respective meal object!

Reply:
okey Markus, let me have a check at it again, thanks a lot